src/conf_mode/system-display.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2020 Francois Mertz fireboxled at gmail.com
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 or later as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import re
+
+from sys import exit
+
+from vyos.config import Config
+from vyos import ConfigError
+from vyos.util import run
+from vyos.template import render
+
+from vyos import airbag
+airbag.enable()
+
+def get_config():
+    c = Config()
+
+    if not c.exists('system display'):
+        return None
+
+    c.set_level('system display')
+
+    return c.get_config_dict([])
+
+def generate(c):
+    if c == None:
+        return None
+    # Render config file for daemon LCDd
+    render('/etc/LCDd.conf', 'system-display/LCDd.conf.tmpl', c)
+    # Render config file for client lcdproc
+    render('/etc/lcdproc.conf', 'system-display/lcdproc.conf.tmpl', c)
+
+    return None
+
+def verify(c):
+    if c == None:
+        return None
+
+    if c.get('model') == None:
```

Comment:
We should follow a common pattern which is `if not 'model' in c.keys()`

Reply:
OK, will do.